### PR TITLE
Feature/marp 763 doc for swagger UI on mp ap is

### PR DESCRIPTION
**Pull request by ntqdinh-axonivy**

### CODE REVIEW

File: marketplace-service/src/main/java/com/axonivy/market/controller/ProductController.java [hunk 1]
```diff
@@ -45,18 +52,33 @@ public class ProductController {
...
     this.assembler = assembler;
     this.pagedResourcesAssembler = pagedResourcesAssembler;
   }
 
-  @Operation(summary = "Find all products", description = "Be default system will finds product by type as 'all'")
   @GetMapping()
-  public ResponseEntity<PagedModel<ProductModel>> findProducts(@RequestParam(name = TYPE) String type,
-      @RequestParam(required = false, name = KEYWORD) String keyword,
-      @RequestParam(name = LANGUAGE) String language, Pageable pageable) {
+  @Operation(summary = "Find all products", description = "Be default system will finds product by type as 'all'", parameters = {
```
Comment (ndkhanh-axonivy) [inline marketplace-service/src/main/java/com/axonivy/market/controller/ProductController.java:63]: Please replace "Find all products" to "Retrieve a paginated list of all products, optionally filtered by type, keyword, and language."

File: marketplace-service/src/main/java/com/axonivy/market/controller/FeedbackController.java [hunk 2]
```diff
@@ -72,25 +90,41 @@ public ResponseEntity<PagedModel<FeedbackModel>> findFeedbacks(@PathVariable(ID)
...
 
   @PostMapping
-  public ResponseEntity<Void> createFeedback(@RequestBody @Valid FeedbackModel feedback,
-      @RequestHeader(value = AUTHORIZATION) String authorizationHeader) {
+  @Operation(summary = "Create user feedback", description = "Save user feedback of product with their token from Github account.")
+  @io.swagger.v3.oas.annotations.parameters.RequestBody(description = "Example request body for feedback", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = FeedbackModel.class)))
+  @ApiResponses(value = {
+          @ApiResponse(responseCode = "201", description = "Successfully created user feedback"),
+          @ApiResponse(responseCode = "401", description = "Unauthorized request")})
+  public ResponseEntity<Void> createFeedback(@RequestBody
```
Comment (ndkhanh-axonivy) [inline marketplace-service/src/main/java/com/axonivy/market/controller/FeedbackController.java:119]:
```
  @Getter
  @Setter
  @Schema(name = "CustomFeedbackModel", description = "Custom model for user feedback")
  public static class CustomFeedbackModel {

    @Schema(description = "ID of the product", example = "adobe-acrobat-connector")
    private String productId;

    @Schema(description = "Content of the feedback", maxLength = 500, example = "This is a great product!")
    private String content;

    @Schema(description = "Rating given by the user", minimum = "1", maximum = "5", example = "4")
    private int rating;
  }
```
  
  Could you please add this model for me to CustomFeedbackModel, we just need 3 property productId, content and rating for it

**Review by ndkhanh-axonivy — COMMENTED** [review]
Please check these comments, thanks

**Review by octopus-axonivy — APPROVED** [review]
LGTM

**Review by ndkhanh-axonivy — APPROVED** [review]
LGTM

**Pull request merged**